common/lib/dependabot/pull_request_creator/labeler.rb

```diff
@@ -4,10 +4,16 @@
 require "dependabot/pull_request_creator"
 module Dependabot
   class PullRequestCreator
-    class Labeler
+    class Labeler # rubocop:disable Metrics/ClassLength
```

Comment:
Not sure if we should try and split or shorten the class to not trigger this violation in this PR